FinancialModelingPrepApi/Abstractions/Model/ICurrentQuote.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Text.Json.Serialization;
+
+namespace MatthiWare.FinancialModelingPrep.Abstractions.Model
+{
+    public interface ICurrentQuote
```

Comment:
No need to have this interface, just create the different models with the properties that they have. 
Futures/Forex Quote models should not be forced to implement the Eps or PE properties